samples/subsys/input/input_dump/prj.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_LOG=y
+CONFIG_LOG_MODE_MINIMAL=y
+CONFIG_GPIO=y
```

Comment:
GPIO should not be here, but selected by appropriate input driver/s

Reply:
Good point, dropped.